02_Data_Management/Layers_and_Table_Views.Rmd

```diff
@@ -1,4 +1,54 @@
-## Select by Location
+## Layers and Table Views toolset
```

Comment:
This suggestion is a consequence of arranging everything by toolbox and toolset. From this point of view, adding "Select by location / atttribute" to "Data Management --> Layers and Table Views" is reasonalbe. However: We are not only describing Geoprocessing tools in this book, but also GUI aspects (Importing data, making maps etc). From this point of view, I think it would make more sense to keep the chapter "select" with two subchapters "select by location" and "select by attribute":